app/src/org/commcare/CommCareApplication.java

```diff
@@ -1476,4 +1478,7 @@ public boolean checkPendingBuildRefresh() {
         return false;
     }
 
+    public PrototypeFactory getPrototypeFactory(Context c) {

```

Comment:
Why is this function necessary? (instead of just calling DbUtil.getPrototypeFactory()